InternalTimeSheet/data/src/main/java/br/com/accera/data/user/datasource/local/UserLocalDataSourceImpl.java

```diff
@@ -0,0 +1,44 @@
+package br.com.accera.data.user.datasource.local;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.util.Log;
+
+import javax.inject.Inject;
+
+import br.com.accera.callback.DataCompleteResponse;
+import br.com.accera.data.user.UserDto;
+
+/**
+ * Created by LuisAlmeida on 28/05/18.
+ */
+
+public class UserLocalDataSourceImpl implements UserLocalDataSource{
+
+    @Inject
+    public UserLocalDataSourceImpl() {
+    }
+
+    @Override
+    public void saveUser(UserDto user, DataCompleteResponse response) {
+        new Thread(new Runnable() {
```

Comment:
It's a mock approach, isn't it? hahaa. We need to implement this feature in the future.